internal/legacy/dockerregistry/registry/set.go

```diff
@@ -24,59 +24,13 @@ import (
 // Various set manipulation operations. Some set operations are missing,
 // because, we don't use them.
 
-// ToSet converts a RegInvImage to a Set.
-func (a RegInvImage) ToSet() container.Set {
```

Comment:
These are unused and (IMO) are not well defined - observe how ToSet "flattens" a map into a set, but the values have meaning...

Reply:
Makes sense; thanks Justin!